dev/io.openliberty.persistence.3.0.thirdparty/resources/org/eclipse/persistence/exceptions/i18n/ConcurrencyExceptionResource.nlsprops

```diff
@@ -43,7 +43,7 @@
 
 2008=Max number of attempts to lock object: {0} exceeded.  Failed to merge the transaction.
 
-2009=Max number of attempts to lock object exceeded.  Failed to build the object. Thread: {0} has a lock on the object but thread: {1} is building the object.
+2009=Max number of attempts to lock object exceeded.  Failed to build the object. Thread: {0} has a lock on the object but thread: {1} is building the object
```

Comment:
Add period after `object` at the end of the sentence. Change back to the message before.